styles.css

```diff
@@ -0,0 +1,958 @@
+}
+
+.footer-links li:hover {
+  color: white;
+}
+
+.secondary_footer {
+  background-color: black;
+  color: white;
+  display: flex;
+}
+
+.heart {
+  color: red;
+}
+
+/* medium screen width less than 1023px: first break point */
+
+@media (max-width: 1023px) and (min-width: 768px) {
+  /* nav bar */
+
+  .scroll-nav {
+    display: grid;
+    grid-template-columns: 1fr 8fr;
+    grid-template-rows: 50px 50px;
+  }
+
+  .nav-right {
+    display: none;
+  }
+
+  .nav-left {
+    grid-column-start: 2;
+    grid-column-end: 3;
+    grid-row-start: 1;
+    grid-row-end: 2;
+  }
+
+  .second-list {
+    grid-column-start: 2;
+    grid-column-end: 3;
+    grid-row-start: 2;
+    grid-row-end: 3;
+  }
+
+  #logo-container {
+    align-items: center;
+    background: linear-gradient(to bottom right, #c09, #f42 85%);
+    display: flex;
+    height: 55px;
+    justify-content: center;
+    width: 15%;
+  }
+
+  /* section one : */
+
+  .first_content {
+    display: grid;
+    grid-template-columns: 2fr 1fr;
+    grid-column-gap: 20px;
+  }
+
+  .item1-class h3,
+  .items-class h3,
+  .items3-class h3 {
+    font-size: 0.8em;
+  }
+
+  .first_article .latest_news {
+    width: 100%;
+  }
+
+  .news_content,
+  .apple_content,
+  .latest_funding_img,
+  .deals_content {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+  }
+
+  .funding_img {
+    width: 240px;
+  }
+
+  .row-1,
+  .row-2,
+  .row-3,
+  .row-4,
+  .row-5,
+  .row-6,
+  .row-7,
+  .row-8 {
+    width: 260px;
+  }
+
+  .row-1 figure img,
+  .row-2 figure img,
+  .row-3 figure img,
+  .row-4 figure img,
+  .row-5 figure img,
+  .row-6 figure img,
+  .row-7 figure img,
+  .row-8 figure img,
+  .apple_content figure img,
+  .deals {
+    width: 260px;
+    height: 200px;
+  }
+
+  .app_sec {
+    grid-template-columns: repeat(2, 1fr);
+  }
+
+  .app_sec .items1 {
+    width: 400px;
+  }
+
+  .app_sec .items1 figure {
+    width: 400px;
+  }
+
+  .app_sec .items1 h3,
+  h4 {
+    color: red;
+  }
+
+  .app_sec_link {
+    width: 400px;
+    text-decoration: none;
+    color: #555;
+  }
+
+  .main-ul {
+    list-style: none;
+  }
+
+  .main-ul li::before {
+    content: "•";
+    color: red;
+    margin: 10px;
+  }
+
+  .main-ul li {
+    border-bottom: 1px solid #555;
+    line-height: 30px;
+  }
+}
+
+/* small screen width less than 769 first break point */
+
+@media (max-width: 768px) {
+  /* nav bar: */
+
+  header {
+    margin-bottom: 0;
+  }
+
+  .grid-nav-wrapper {
+    display: none;
+  }
+
+  .second-list {
+    height: 72px;
+  }
+
+  #nav-container {
+    display: grid;
+    grid-template-columns: 1fr 8fr;
+    grid-template-rows: 50px;
+  }
+
+  #main-nav {
+    background: #fff;
+    border-bottom: 1px solid #eee;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    position: fixed;
+    top: 0;
+    width: 100%;
+  }
+
+  #logo-container {
+    width: 100px;
+  }
+
+  ul.second-list {
+    display: none;
+  }
+
+  #menu-icon {
```

Comment:
**[REQUIRED]** The menu icon should appear on the right side of the navigation bar. We should have the search icon and menu icon on the left, TNW logo on the right for smaller screens